yellow-paper/docs/decentralization/actors.md

```diff
@@ -0,0 +1,134 @@
+
+### Actor Types
+
+1. **Retail**:
+   - Least knowledgeable and least interested in technical details.
+   - Prefers simplicity and ease of use.
+   - Likely to avoid running any node software or only running if bundled
+2. **Enthusiast**: 
+   - More knowledgeable and interested in technology than retail.
+   - Willing to invest time and resources but not at a professional level.
+3. **Power Users**: 
+   - Technically proficient and deeply engaged.
+   - Likely to have the resources and motivation to run more demanding nodes.
+4. **Developers**: 
+   - Highly knowledgeable with technical skills.
+   - Interested in detailed state and historical data for development and testing.
+5. **Self-Validators**: 
+   - Aim to participate actively in keeping the blockchain available.
+   - Interested in relying on their own nodes and resources (Archive Nodes).
+   - Likely to run nodes that contribute directly to the blockchain's operation, like Sequencers or Provers.
+6. **Institutions**: 
+   - Have significant resources (and potentially technical) capabilities.
+   - Interested in robust participation, potentially running Sequencer or Provers for financial gain.
+
+:::info State-Snapshots
+State snapshots is likely made and published by actors in the **Self-validator** or **Institution** category. These are actors that have the resources to run Archive Nodes and are interested in the historical data.
+:::
+
+### Private Execution Environment (PXE)
+
+The Aztec PXE is required to do anything meaningful for an end-user. It is the interface between the user and the Aztec node. The PXE is a private enclave that runs on the end-user's machine.
+
+It is responsible for execution contracts and building proofs for the user's transactions privately. This requires the PXE to be connected to a node that can provide the user's state.
+
+:::info Note-discovery
+Note, that as the node will not store secrets for the user. The user relies on some mechanism to figure out "what to lookup", this is what is sometimes referred to as note-discovery: discovering what notes belong to the user.
+:::
+
+As the PXE learns of the users notes, it is stored in the PXE's storage for later usage. Beware that the PXE need to revalidate the membership paths of the notes when they are to be spent which will connect to the node.
+
+
+### Users
+
+From the end-users perspective, they will be interacting with the PXE, which is then talking to the aztec node and the underlying chain.
+
+```mermaid
+graph LR
+   A[User] --> B[PXE]
+   B --> C[Aztec Node]
+   C --> D[Ethereum Node]
+```
+
+The type of node the PXE is connected to depends on the actor type. 
+
+:::info Bundling
+For a mobile application to work nicely for the users the PXE will most likely be bundled with a light node on device. This will allow the user to interact with the system without wider knowledge. Settings might allow you to connect the PXE directly to an endpoint for a node for advanced users.
```

Comment:
```suggestion
For a mobile application to work smoothly and for it to not leak information about which note hashes in the note hash tree pertain to notes owned by the users the PXE will most likely be bundled with a light node on device. This will allow the user to interact with the system without wider knowledge. In the settings we might allow you to connect the PXE directly to an external node (this might be relevant for advanced users).
```